### PR TITLE
Allow poll question answers to be editable

### PR DESCRIPTION
**Pull request by aitbw**
Where
=====
* **Related Issue:** #1992
* **Related PR:** #1977

What
====
- Allow poll question answers to be editable

How
===
* Adapted/created views/forms according to the new design
* Adapted the controller to fit the new feature
* Appended the necessary routes
* Added translated labels in English, Spanish and French
* Wrote tests for the feature

Screenshots
===========
* See mockups on #1992

Test
====
* Increased coverage for Create and Update scenarios

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -300,7 +300,7 @@
         end
       end
 
-      resources :questions do
+      resources :questions, shallow: true do
         resources :answers, except: [:index, :destroy, :delete], controller: 'questions/answers', shallow: true do
```
Comment (aitbw) [inline config/routes.rb]: I'm pretty sure we don't need `shallow: true` here if we have it specified on `resources :questions`

Comment (voodoorai2000) [reply config/routes.rb]: 👍

File: app/controllers/admin/poll/questions/answers_controller.rb [hunk 2]
```diff
@@ -44,7 +50,4 @@ def load_answer
       @answer = ::Poll::Question::Answer.find(params[:id] || params[:answer_id])
     end
 
-    def load_answer
```
Comment (aitbw) [inline app/controllers/admin/poll/questions/answers_controller.rb]: Why is this being deleted but the `before_action` stays? We need this to show, edit and update the answer when requested.

Comment (voodoorai2000) [reply app/controllers/admin/poll/questions/answers_controller.rb]: It's still there, it was duplicated
https://github.com/wairbut-m2c/consul/blob/23ef9ded1d02dcbf4304bd8eb89cd304d0f29156/app/controllers/admin/poll/questions/answers_controller.rb#L49

File: config/routes.rb [hunk 3]
```diff
@@ -301,7 +301,7 @@
       end
 
       resources :questions, shallow: true do
-        resources :answers, except: [:index, :destroy, :delete], controller: 'questions/answers', shallow: true do
+        resources :answers, except: [:index, :destroy], controller: 'questions/answers', shallow: true do
```
Comment (aitbw) [inline config/routes.rb]: I don't think we need the `shallow: true` here if we have it declared on `resources :questions`

File: app/controllers/admin/poll/questions/answers_controller.rb (hunk 2 above)
Comment (aitbw) [reply app/controllers/admin/poll/questions/answers_controller.rb]: Oh, sorry about that.

Comment (voodoorai2000) [reply app/controllers/admin/poll/questions/answers_controller.rb]: no problem, appreciated it!

File: config/routes.rb (hunk 3 above)
Comment (voodoorai2000) [reply config/routes.rb]: 👍

**Pull request merged**